profiles/cclcc/hud/extramenus.lua

```diff
@@ -2,19 +2,39 @@ root.ExtraMenus = {
     ClearListMenu = {
         DrawType = DrawComponentType.SystemMenu,
         Type = ClearListMenuType.CCLCC,
-        BackgroundSprite = "ClearListBackground",
+        BackgroundSprite = "MenuBackground",
         FadeInDuration = 0.2,
         FadeOutDuration = 0.2,
         ClearListBookLayerSprite = "ClearListBookLayer",
         ClearListGuideSprite = "ClearListGuide",
         ClearListGuideX = 0,
         ClearListGuideY = 990,
-        ClearListMaskSprite = "ClearListMask"
+        LibraryMaskSprite = "LibraryMask",
+    },
+    LibraryMenu = {
+        DrawType = DrawComponentType.SystemMenu,
+        Type = LibraryMenuType.CCLCC,
+        BackgroundSprite = "MenuBackground",
+        FadeInDuration = 0.2,
+        FadeOutDuration = 0.2,
+        LibraryBackgroundSprite = "LibraryBackground",
+        LibraryBackgroundPosition = { X = 291, Y = 0 },
+        LibraryIndexSprite = "LibraryIndex",
+        LibraryIndexPosition = { X = 0, Y = 31 },
+        LibraryButtonGuideSprite = "LibraryButtonGuide",
+        LibraryButtonGuidePosition = { X = 0, Y = 988 },
+        LibraryMaskSprite = "LibraryMask",
     }
 }
 
+-- Common
+root.Sprites["LibraryMask"] = {
+    Sheet = "MenuChip",
+    Bounds = { X = 154, Y = 140, Width = 1900, Height = 1061 },
+};
+
 -- ClearList
-root.Sprites["ClearListBackground"] = {
+root.Sprites["MenuBackground"] = {
```

Comment:
I don't think clearlist should actually have background sprite from the titlemenu, titlemenu is supposed to be rendered while clearlist is